website/content/docs/upgrading/upgrade-to-1.17.x.mdx

```diff
@@ -81,6 +81,14 @@ Users may not be able to log into Vault if the JWT role is configured
 incorrectly. For additional details, refer to the
 [JWT auth method (API)](/vault/api-docs/auth/jwt) documentation.
 
+### Auto-rolled billing start date
+
+ As of 1.17.3 and later, the billing start date (license start date if not configured) rolls over to the latest billing year at the end of the last cycle i.e; billing start dates from 
+ earlier years will get updated to current calendar year, unless the date has not yet occured this year, in which case
```

Comment:
I will move this part that explains auto rolling to a common mdx and link it here if this description is okay